webservice/template/admin/faq/edit.tmpl

```diff
@@ -9,7 +9,7 @@
     <h2 class="display-4 mb-3" style="float:left;">Frequently Asked Questions <a href="/admin/faq"
                                                                                  class="btn btn-primary btn-sm px-3">Back</a>
     </h2>
-    <p class="text-right" style="margin-top: 20px;"><strong>Last Updated:</strong> {{ PRETTYTIME .Content.Date }}</p>
+    <p class="text-right" style="margin-top: 20px;"><strong>Last Updated:</strong> {{ PRETTYTIME .Content.Date }} <i>(Norwegian time!)</i></p> <!-- TODO time-norwegian -->
```

Comment:
Remove "(Norwegian time!)"

Reply:
Christopher asked for this specific, so nah